includes/core-changes/aspnetcore/5.0/signalr-usesignalr-useconnections-removed.md

```diff
@@ -0,0 +1,47 @@
+### SignalR: UseSignalR and UseConnections methods removed
+
+In ASP.NET Core 3.0, SignalR adopted endpoint routing. As part of that change, the <xref:Microsoft.AspNetCore.Builder.SignalRAppBuilderExtensions.UseSignalR%2A>, <xref:Microsoft.AspNetCore.Builder.ConnectionsAppBuilderExtensions.UseConnections%2A>, and some related methods were marked as obsolete. In ASP.NET Core 5.0, those obsolete methods were removed. For the full list of methods, see [Affected APIs](#affected-apis).
```

Comment:
It would be great to be able to link to the include for the 3.0 breaking change for these methods, but I don't think that would be possible since the 3.1-5.0 file wouldn't have that breaking change. Perhaps you can just add in the same code examples that are in the 3.0 breaking change to the "Recommended action" section.

Reply:
Great suggestion!